reana_server/validation.py

```diff
@@ -146,7 +146,8 @@ def validate_retention_rule(rule: str, days: int) -> None:
     if ".." in rule_path.parts:
         raise REANAValidationError(f"Retention rule {rule} cannot contain '..'")
 
-    if days >= WORKSPACE_RETENTION_PERIOD:
+    workspace_rules_are_not_disabled = WORKSPACE_RETENTION_PERIOD != -1
```

Comment:
*question(non-blocking):* unfortunately, I am not sure how not to repeat `WORKSPACE_RETENTION_PERIOD != -1` check in two functions. One idea is to make it a separate variable in `config.py`. Not sure if it is worth it. WDYT?